index.js

```diff
@@ -183,6 +226,19 @@ function init (options) {
   }
 }
 
+/**
+ * @typedef {Object} Options
+ * @prop {string} [base] Path to package.json to import aliases from.
+ */
```

Comment:
AFAIK someone made a good job with defining types in `@types/module-alias` package, so I would remove this since it only scratches the surface, but not defining all Typescript types for the code.